tests/Unit/IO/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ set(LIBRARY_SOURCES
   Test_VolumeData.cpp
   )
 
+add_subdirectory(DataImporter)
```

Comment:
Why are you adding this in the middle? It's creating it's own library so add it at the front or the end.